include/brickengine/input.hpp

```diff
@@ -310,6 +444,9 @@ class BrickInput {
         // Mouse input
         keycode_mapping.insert({SDL_BUTTON_LEFT, InputKeyCode::EKey_mouse_left});
         keycode_mapping.insert({SDL_BUTTON_RIGHT, InputKeyCode::EKey_mouse_right});
+
+        // Controllers
```

Comment:
I think you meant to put this comment after line 321 (void generateInputs() {) instead of here?